src/tribler-core/tribler_core/components/tag/community/tag_community.py

```diff
@@ -55,36 +57,38 @@ def request_tags(self):
 
         peer = random.choice(self.get_peers())
         self.requests.register_peer(peer, REQUESTED_TAGS_COUNT)
-        self.logger.debug(f'Request {REQUESTED_TAGS_COUNT} tags')
+        self.logger.info(f'-> request {REQUESTED_TAGS_COUNT} tags from peer {hexlify(peer.mid)}')
```

Comment:
nit: `hexlify` is not necessary anymore, we can use `hex` method of `bytes` object instead:
https://docs.python.org/3/library/stdtypes.html#bytes.hex

```suggestion
        self.logger.info(f'-> request {REQUESTED_TAGS_COUNT} tags from peer {peer.mid.hex()}')
```
(this comment is also applied to all other places where `hexlify` used in this PR)

Reply:
Fixed